packages/components/src/dropdown/index.js

```diff
@@ -43,11 +43,14 @@ class Dropdown extends Component {
 	 * When contents change height due to user interaction,
 	 * `refresh` can be called to re-render Popover with correct
 	 * attributes which allow scroll, if need be.
+	 * @deprecated
 	 */
 	refresh() {
-		if ( this.popoverRef.current ) {
```

Comment:
This `popoverRef` is now unused. Can we remove it? 

- The dependency on `createRef`
- The assignment in `constructor`
- The `ref` prop in `render`

Reply:
Done in 3b34fc47606f448ac642108e0f2436baf55d3b56